### PR TITLE
Docs: update the hyper link for latest release in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by sayboras**
### Description

As per below screen shot, the link is pointing to master, upon clicking it returns 404.

image

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `site/content` directory)
- [ ] Manually tested
- [ ] Added labels for change area (e.g. `area/nodegroup`), target version (e.g. `version/0.12.0`) and kind (e.g. `kind/improvement`)
- [ ] Make sure the title of the PR is a good description that can go into the release notes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -319,6 +319,6 @@ You want to keep an eye on Circle CI for the progress of the release ([0.3.1 exa
 
 ### Latest release
 
-To get the latest release you can use the link [https://github.com/weaveworks/eksctl/releases/latest]().
+To get the latest release you can use the link [https://github.com/weaveworks/eksctl/releases/latest](https://github.com/weaveworks/eksctl/releases/latest).
```
Comment (dholbach) [inline CONTRIBUTING.md:322]:
```suggestion
To get the latest release you can use the link <https://github.com/weaveworks/eksctl/releases/latest>.
```

Comment (dholbach) [reply CONTRIBUTING.md:322]: Thanks a lot for fixing the link. This might be a more compact way of writing it.

Comment (sayboras) [reply CONTRIBUTING.md]: Oh thanks, I didn't know about this < > :+1:

Comment (sayboras) [reply CONTRIBUTING.md]: @dholbach I noticed that you have something like link checker in flagger, any plan to bring those checks to eksctl as well ?

Comment (dholbach) [reply CONTRIBUTING.md]: Yes. So we have link checkers for `flux` and `wksctl` and are going to land them for `ignite` and `helm-operator` too. Sounds like a good idea!

Comment (sayboras) [reply CONTRIBUTING.md]: :100: 

I mentioned flagger wrong haha, i noticed about link checker as one of my pr in flux failed :+1:,  then I took a look and found out you have implemented very extensive link checkers :100: :medal_military:

**Pull request merged**